src/ui/Forms/AudioToText/WhisperModelDownload.cs

```diff
@@ -108,9 +108,10 @@ private void MultiFileDownload()
                     var pct = (int)Math.Round(progress * 100.0, MidpointRounding.AwayFromZero);
                     labelPleaseWait.Text = LanguageSettings.Current.General.PleaseWait + "  " + pct + "%";
                 });
-                foreach (var url in LastDownloadedModel.Urls)
+
+                using (var httpClient = DownloaderFactory.CreateProxiedHttpClient())
                 {
-                    using (var httpClient = DownloaderFactory.MakeHttpClient())
```

Comment:
This line was the issue* as it's creating a new instance of http-client for every url!